vars/withPipeline.groovy

```diff
@@ -14,7 +14,6 @@ import uk.gov.hmcts.contino.PipelineCallbacksConfig
 import uk.gov.hmcts.contino.PipelineCallbacksRunner
 import uk.gov.hmcts.pipeline.AKSSubscriptions
 import uk.gov.hmcts.pipeline.TeamConfig
-import uk.gov.hmcts.pipeline.DeprecationConfig
```

Comment:
this needs to stay - please add back in

Reply:
The `DeprecationConfig` function is not referenced in the file hence I removed it. Can I know why it needs to stay

Reply:
add it back in